partiql-lang/src/test/kotlin/org/partiql/lang/planner/transforms/PartiQLSchemaInferencerTests.kt

```diff
@@ -418,6 +418,36 @@ class PartiQLSchemaInferencerTests {
                     )
                 )
             ),
+            SuccessTestCase(
```

Comment:
These two tests are under the `selectStar` testing function. Perhaps they should be added in a different testing function or their own testing function?